### PR TITLE
Update staleness filters to default to all values

### PR DESCRIPTION
**Pull request by kruai**
# Overview

This PR is being created to address [ESSNTL-1138](https://issues.redhat.com/browse/ESSNTL-1138).
Updates the "staleness" filter to default to all values in the enum, instead of leaving out the "stale_warning" value.
This is necessary to address [this bug](https://issues.redhat.com/browse/ESSNTL-728). An additional UI change will be required to fully fix the issue, but since the UI always sends us a staleness filter right now, this change should not affect anything on its own.

## PR Checklist

- [x] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: tests/test_xjoin.py [hunk 1]
```diff
@@ -822,6 +823,12 @@ def test_tags_query_variables_default_staleness(
                             "lte": "2019-12-16T10:10:06.754201+00:00",
                         }
                     },
+                    {
+                        "stale_timestamp": {
+                            "gt": "2019-12-02T10:10:06.754201+00:00",
+                            "lte": "2019-12-09T10:10:06.754201+00:00",
```
Comment (thearifismail) [inline tests/test_xjoin.py:829]: Should include timestamp value using the special UTC designator ("Z"), like "2019-12-09T10:10:06.75Z"

**Review by thearifismail — COMMENTED** [review]
Commented

File: tests/test_xjoin.py (hunk 1 above)
Comment (kruai) [reply tests/test_xjoin.py:829]: Nope, doesn't need it here @thearifismail - look at all the other timestamps in this test (and in this file)

**Review by thearifismail — APPROVED** [review]
LGTM

**Comment by ryandillinfelton** [comment]
@kruai have you checked with front end to get an idea of how this will effect them? I don't suppose it would be a problem, but it seems possible.

**Comment by kruai** [comment]
> @kruai have you checked with front end to get an idea of how this will effect them? I don't suppose it would be a problem, but it seems possible.

@ryandillinfelton I've tagged Ben Clark on the work a couple of times, but haven't gotten a response yet. From my manual testing, though, I don't see this affecting the HBI UI at all yet. The UI adds "fresh" and "stale" filters as a default, which they send manually in the `/hosts` requests. It also sends those 2 default filters when *no* staleness filters are selected, which I created [this Jira](https://issues.redhat.com/browse/ESSNTL-1139) to fix.

**Comment by kruai** [comment]
Converted this to a draft until the requirements are ironed out.

**Comment by kruai** [comment]
The discussion around the requirements has faded, and nobody else has complained about the "bug". I'm closing this until someone confirms that this is actually the change they want.

**Pull request closed without merging**